README.md

```diff
@@ -22,24 +22,18 @@ __default__
 ```yaml
 ---
 extends: braintree
-rules:
-  locally-overriden-rule: 1
 ```
 
 __browserify__
 ```yaml
 ---
 extends: braintree/client
-rules:
```

Comment:
Why are we removing examples for how to override the config? Is this not possible with eslint@v4?

Reply:
@crookedneighbor I can't actually find any documentation on `locally-overriden-rule`, and I get an error when using it that it's an undefined property. To override rules you just add them to your local rules, overriding any options if they were set in a parent as well.

Reply:
This is a readme to show how to override rules. `locally-overriden-rule` is not a real rule, just an example for how you would do it with a real rule. 

If that's confusing, we could have a real rule here.

Reply:
A real rule wouldn't be bad here

Reply:
or something like `example-locally-overriden-rule`